CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ Bug fixes:
 
 - [#491 Remove redundant Google Analytics](https://github.com/alphagov/govuk_prototype_kit/pull/491)
 
+Unreleased:
```

Comment:
Sorry, just spotted there's already an unreleased heading. Do you consider this a new feature?

Reply:
I guess this is a drawback of having the kit and the site be the same thing. I think in the past we've called new things on the site 'features' so this is consistent.